bin/python/license_manager_script.py

```diff
@@ -0,0 +1,108 @@
+# encoding: utf-8
+# author:   Jan Hybs
+
+
+import sys
+import pathfix
+
+pathfix.append_to_path()
+
+import system.versions
+
+system.versions.require_version_2()
+
+from optparse import OptionParser
+from utils.license_manager import LicenseManager
+
+
+def create_parser():
+    """Creates command line parse"""
+    parser = OptionParser()
+    parser.add_option("-l", "--license", dest="license_path", metavar="FILENAME", default=None,
+                      help="Absolute or relative path to file containing new license,"
+                           " if not specified, license will be only removed")
+    parser.add_option("-f", "--file", dest="files", metavar="FILENAME", default=[], action="append",
+                      help="Absolute or relative path to file which will be processed")
+    parser.add_option("-d", "--dir", dest="dirs", metavar="DIRNAME", default=[], action="append",
+                      help="Absolute or relative path to dir which will be recursively processed")
+
+    parser.add_option("-g", "--git", dest="git", metavar="DIRNAME", default=None,
+                      help="Location of git repository (will add additional information to license)")
```

Comment:
license file supports placeholders, when script is called with valid git root ```--git git-root-dir-path```, placeholders offer some git related info:
```python
"""
This is license for file '{filename}' in branch {branch}, last edited by {last_author}
License goes here
"""
```
will produce something like:
```
This is license for file 'main.cc' in branch master, last edited by Jan Brezina
License goes here
```